client/src/partials/finance/cashflow/cashflow.html

```diff
@@ -0,0 +1,239 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
+      <li class="title">
+        {{ "CASHFLOW.TITLE" | translate }}
+        <b ng-show="ReportCtrl.state !== 'generate'" class="label label-default">
+          <span class="fa fa-cogs"></span> {{ "REPORT.CONFIGURATION" | translate }}
+        </b>
+        <b ng-show="ReportCtrl.state === 'generate'" class="label label-info">
+          <span class="fa fa-check-circle-o"></span> {{ "REPORT.GENERATED" | translate }}
+        </b>
+      </li>
+    </ol>
+
+    <div ng-show="ReportCtrl.state === 'generate'" class="toolbar">
+      <div class="toolbar-item btn-group btn-group-sm">
+        <button type="button" class="btn btn-default" ng-click="ReportCtrl.print()">
+          <i class="fa fa-print"></i> {{ 'FORM.BUTTONS.PRINT' | translate }}
+        </button>
+        <button type="button" class="btn btn-default" ng-click="ReportCtrl.reconfigure()">
+          <i class="fa fa-repeat"></i>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="flex-content">
+  <div class="container-fluid">
+
+    <div ng-switch="ReportCtrl.state">
+      <div class="row" ng-switch-default>
+
+        <!-- configuration panel  -->
+        <div class="col-xs-6 col-xs-offset-3 panel panel-default">
+          <div class="panel-body">
+            <form name="ConfigForm" bh-submit="ReportCtrl.generate()" bh-form-defaults novalidate>
+
+              <!-- Date interval  -->
+              <bh-date-interval
+                date-from="ReportCtrl.dateFrom"
+                date-to="ReportCtrl.dateTo">
+              </bh-date-interval>
+
+              <!-- cashbox selection  -->
+              <div class="form-group">
+                <label class="control-label">
+                  <span class="fa fa-briefcase"></span> {{ 'FORM.SELECT.CASHBOX' | translate }}
+                </label>
+                <select
+                  class="form-control"
+                  name="cashbox"
+                  ng-model="ReportCtrl.cashbox"
+                  ng-options="cash as cash.label + ' ' + cash.symbol for cash in ReportCtrl.cashboxes"
```

Comment:
I would pre-format this in your controller.  Something like:
```js
Cashboxes.read()
  .then(function (cashboxes) {
    cashboxes.forEach(function (cashbox) {
      cashbox.hrlabel = cashbox.label + '  ' + cashbox.symbol;
    });
    vm.cashboxes = cashboxes;
    // etc ...
  });
```
... then ...
```html
<select
  ng-options="cash as cash.hrlabel for cash in ReportCtrl.cashboxes"
  ...>
```

It's a small optimization, and this won't block your PR.